### PR TITLE
new ruleset for letterboxd

### PR DESCRIPTION
**Pull request by phibo23**
https officially supported since a few days ago according to their feedback thread http://feedback.letterboxd.com/forums/136771-general/suggestions/2505918-login-registration-over-https

### CODE REVIEW

**Comment by phibo23** [comment]
that file src/chrome/content/rulesets.json was created when running
```
test/firefox.sh --justrun
```
so I figured it should be added to gitignore

**Comment by J0WI** [comment]
You should also check `*.ltrbxd.com`.
The are at least `a` and `s2`

**Comment by J0WI** [comment]
Sorry, I closed accidentally.

**Comment by phibo23** [comment]
ok, they have a wildcard certificate for *.ltrbxd.com (just as they do for letterboxd.com) but not all servers of their cdn (?) appear to be configured for https. I'll look into it and add anything I find to be working.

**Comment by phibo23** [comment]
as far as I understand, all of the servers of their cdn support https but they don't have certificates for multi level subdomains yet. If you visit letterboxd.com without https, static files are served from a variety of cdn servers. But if you visit with https, only the three single level subdomains (a, s1 and s2) are used, so I guess it's their own decision to "bypass" the cdn as a sideeffect of using https.

File: src/chrome/content/rules/Letterboxd.com.xml [hunk 1]
```diff
@@ -7,6 +7,15 @@
 <!-- 	<target host="embed.letterboxd.com" /> -->
 <!-- 	<target host="feedback.letterboxd.com" /> -->
 
+<!-- there is also ltrbxd.com which appears to be mainly used for a cdn. However not all subdomains work correctly with https. For example, every file on a.ltrbxd.com is also available on skyfall.a.ltrbxd.com, oblivion.a.ltrbxd.com etc. These servers can be accessed with https but browsers will display errors/warnings because there is only a wildcard certificate for *.ltrbxd.com and not one for each of the subdomains. -->
```
Comment (J0WI) [inline src/chrome/content/rules/Letterboxd.com.xml]: Please use a linebreak here

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**